www/fram05/semestralni_prace_2/index.js

```diff
@@ -0,0 +1,55 @@
+var checked_chars = "";
+function check_checks() {
+    var capitals_checked = "";
+    var small_checked = "";
+    var numbers_checked = "";
+    var others_checked = "";
+
+    if (document.form.capitals.checked == false && document.form.smalls.checked == false && document.form.numbers.checked == false && document.form.others.checked == false) {
+        window.alert("None of the parameters were selected. Please choose at least one to process the request!");
+    } else {
+        if (document.form.capitals.checked == true) {
+            capitals_checked = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        } else {
+            capitals_checked = "";
+        }
+        if (document.form.smalls.checked == true) {
+            small_checked = "abcdefghijklmnopqrstuvwxyz";
+        } else {
+            small_checked = "";
+        }
+        if (document.form.numbers.checked == true) {
+            numbers_checked = "123456789";
+        } else {
+            numbers_checked = "";
+        }
+        if (document.form.others.checked == true) {
+            others_checked = "@/\()-*+#&";
+        } else {
+            others_checked = "";
+        }
+    }
+
+    checked_chars = capitals_checked.concat(small_checked, numbers_checked, others_checked);
+
+
+    document.getElementById("show").innerHTML = checked_chars;
+}
+
+function generate_pswd() {
+    var chars = document.getElementById("show").innerHTML;
```

Comment:
mnozinu pouzitych znaku pro vytvoreni hesla je lepsi mit v nejake promenne a nebrat to z DOMU